apps/climatemappedafrica/src/lib/data/blockify/index.js

```diff
@@ -1,20 +1,24 @@
+import explorePage from "./explore-page";
 import hero from "./hero";
 import pageHero from "./page-hero";
 import team from "./team";
+import tutorial from "./tutorial";
 
 const propsifyBlockBySlug = {
+  "explore-page": explorePage,
   hero,
   "page-hero": pageHero,
   team,
+  tutorial,
 };
 
-export const blockify = async (blocks, api, context) => {
+export const blockify = async (blocks, api, context, hurumap) => {
   const promises = blocks?.map(async (block) => {
     const slug = block.blockType;
     const propsifyBlock = propsifyBlockBySlug[slug];
 
     if (propsifyBlock) {
-      return propsifyBlock(block, api, context);
+      return propsifyBlock(block, api, context, hurumap);
```

Comment:
This "overload" is risky I think perhaps we should use an object instead to avoid what we have done in `blockify/hero.js`

Reply:
@koechkevin can you clarify more on: 

> we should use an object instead to avoid what we have done in blockify/hero.js

Reply:
I meant
```suggestion
      return propsifyBlock({ block, api, context, hurumap });
```

Reply:
Aaah, named params. Great idea.